src/raster/r.forcircular/r.forcircular.html

```diff
@@ -0,0 +1,278 @@
+  hf_noslope="tractor" c_noslope="tractor" ind1=indicator1 ind2=indicator2 ind3=indicator3 \
+  ind4=indicator4 ind5=indicator5 ind6=indicator6 ind7=indicator7
+</pre></div>
+
+<p>
+The output maps are: technical_surface (forest surface where exploitation is
+technically possibile), economic_surface (forest surface where the exploitation
+is economically advantageous), emission (CO<sub>2</sub> produced by the
+exploitation), annual_avoided_emission (emissions avoided compared to the use of
+fossil fuels), revenues (map of economic revenues), dip (distance from ideal,
+the assessment of the circular economy level of the forest supply chain).
+Below there are two examples of the output maps corresponding respectively to
+the
+stumpage value and the annual avoided emission: <br><br>
+<div align="center" style="margin: 10px">
+    <a href="stumpage.png">
+        <img src="stumpage.png" width="400" alt="stumpage value map" border="0">
+    </a><a href="./avoided_emission.png">
+        <img src="./avoided_emission.png" width="400" alt="Avoided emission map"
+         border="0">
+    </a><br>
+    <i>Examples of output maps: stumpage value and avoided emission</i>
+</div>
+The output text results are the follow:
+<div class="code"><pre>
+#############################
+End of process 
+Name of output maps: 
+rep_roundwood -> roundwood (m3/y): 8507.3917
+rep_timber -> timber pole (m3/y): 195.4894
+rep_firewood -> firewood (m3/y): 7239.1150
+rep_bioenergy -> bioenergy (MWh/y): 6354.3441
+rep_sum_a_stumpage_value -> annual stumpage value (EUR/y): 147608.5463
+rep_ave_stumpage_value -> average stumpage value (EUR/ha): 3998.2780
+rep_ave_a_stumpage_value -> average annual stumpage value (EUR/ha*y-1): 28.2872
+rep_annual_avoided_emission -> annual avoided emissions (t): 1963.9961
+---------------------------
+rep_indicator1 -> annual value of wood on annual yield (euro/m3): 7.2534
+rep_indicator2 -> carbon dioxide emission per cubic meter (t/m3): 0.0001
+rep_indicator3 -> general index of forest surface utilization (ha/y):157.192
+rep_indicator4 -> general index of re-use (m3*y) sum: 292481.3438
+rep_indicator5 -> potential value of wood on real value (euro/euro): 1.3093
+rep_indicator6 -> percentual of wood residuals used in bioenergy production (%): 0.1277
+rep_indicator7 -> avoided CO2 per unit of energy produced (gCO2 /kWh): 308.6331
+dist_tot -> AMC map: 3.7276
+#############################
+</pre></div>
+
+
+<h2>REFERENCE</h2>
+
+Paletto A, Becagli C, Geri F, Sacchelli S, De Meo I. Use of Participatory
+Processes in Wood Residue Management from a Circular Bioeconomy Perspective: An
+Approach Adopted in Italy. Energies. 2022; 15(3):1011.
+<a href="https://doi.org/10.3390/en15031011">
+    https://doi.org/10.3390/en15031011</a>
+
+<h2>SEE ALSO</h2>
+
+<em>
+    <a href="r.green.biomassfor.html">r.green.biomassfor</a>
+</em>
+
+<h2>AUTHORS</h2>
+
+Francesco Geri and Sandro Sacchelli
```

Comment:
Another case with file ending without newline. 
```suggestion
Francesco Geri and Sandro Sacchelli

```